src/cascadia/TerminalSettingsEditor/Utils.h

```diff
@@ -51,29 +51,29 @@
 // of EnumEntries so that we may display all possible values of the given
 // enum type and its localized names. It also provides a getter and setter
 // for the setting we wish to bind to.
-#define GETSET_BINDABLE_ENUM_SETTING(name, enumType, settingsModelName, settingNameInModel)                                                              \
-public:                                                                                                                                                  \
-    winrt::Windows::Foundation::Collections::IObservableVector<winrt::Microsoft::Terminal::Settings::Editor::EnumEntry> name##List()                     \
-    {                                                                                                                                                    \
-        return _##name##List;                                                                                                                            \
-    }                                                                                                                                                    \
-                                                                                                                                                         \
-    winrt::Windows::Foundation::IInspectable Current##name()                                                                                             \
-    {                                                                                                                                                    \
-        return winrt::box_value<winrt::Microsoft::Terminal::Settings::Editor::EnumEntry>(_##name##Map.Lookup(settingsModelName().settingNameInModel())); \

```

Comment:
The only change here is that I removed the `()` from `settingsModelName`. Needed this because I didn't want to expose a getter to `_globals` in `GlobalSettingsViewModel`.